src/platform/backends/shared/linux/process.cpp

```diff
@@ -31,7 +31,9 @@ mp::Process::Process(std::unique_ptr<mp::ProcessSpec>&& spec) : process_spec{std
     if (!process_spec->working_directory().isNull())
         setWorkingDirectory(process_spec->working_directory());
 
-    setProcessChannelMode(QProcess::ForwardedErrorChannel); // default to forwarding child stderr to console
+    QObject::connect(this, &Process::readyReadStandardError, [this]() {
+        mpl::log(mpl::Level::error, qPrintable(process_spec->program()), qPrintable(readAllStandardError()));
```

Comment:
Add `TODO` please to handle newlines better here, so that it doesn't get chunked together?

Reply:
Done